run_project_tests.py

```diff
@@ -1553,6 +1553,11 @@ def get_version(t: ToolInfo) -> str:
 
         return f'{exe} (unknown)'
 
+    if not report:
+        for tool in tools:
+            get_version(tool)
+        return
+
```

Comment:
This has bothered me for a while!

Reply:
Cherry-picked into master, so we don't need to wait until this PR leaves draft mode. :)